sources/api/netdog/src/main.rs

```diff
@@ -58,6 +61,8 @@ static CURRENT_IP: &str = "/var/lib/netdog/current_ip";
 static KERNEL_CMDLINE: &str = "/proc/cmdline";
 static PRIMARY_INTERFACE: &str = "/var/lib/netdog/primary_interface";
 static DEFAULT_NET_CONFIG_FILE: &str = "/var/lib/bottlerocket/net.toml";
+static PRIMARY_SYSCTL_CONF: &str = "/etc/sysctl.d/10-primary_interface.conf";
```

Comment:
Ideally this would sort after `/usr/lib/sysctl.d/50-default.conf` which is where the upstream defaults come from, and after `/usr/lib/sysctl.d/80-release.conf` which is where our defaults come from.

Otherwise the behavior might change when `systemd-sysctl` is run with just this snippet, vs. with all snippets.

```suggestion
static PRIMARY_SYSCTL_CONF: &str = "/etc/sysctl.d/90-primary_interface.conf";
```